src/Raven.Server/NotificationCenter/Handlers/ProxyWebSocketConnection.cs

```diff
@@ -21,11 +21,12 @@ public class ProxyWebSocketConnection : IDisposable
         private readonly ClientWebSocket _remoteWebSocket;
         private readonly WebSocket _localWebSocket;
         private readonly string _nodeUrl;
+        private readonly string _localNodeTag;
         private readonly IMemoryContextPool _contextPool;
         private Task _localToRemote;
         private Task _remoteToLocal;
 
-        public ProxyWebSocketConnection(WebSocket localWebSocket, string nodeUrl, string websocketEndpoint, IMemoryContextPool contextPool, CancellationToken token)
+        public ProxyWebSocketConnection(WebSocket localWebSocket, string localNodeTag, string nodeUrl, string websocketEndpoint, IMemoryContextPool contextPool, CancellationToken token)
```

Comment:
this file shouldn't be changed

Reply:
It needs the local (source) noteTag because it uses ```ReplicationUtils.GetTcpInfo``` that uses it too

Reply:
Removed